### PR TITLE
Modify the docs on scope schemes

### PR DESCRIPTION
**Pull request by Chamindu36**
**[Please don’t  merge  Until WUM fix is released]**

## Purpose
Change the description of adding oauth2 scopes using an open API swagger definition.

## Goals
Fixes: https://github.com/wso2/product-apim-tooling/issues/271

## Approach
Previously it stated that name of the security scheme when adding scopes should be "default". But with the newest fix, this name can be an arbitrary one. This will state that change in docs.
When adding user roles for scopes there is the standard keyword to use. That also stated in here.

image

## User stories
Users can follow the steps to create API using Swagger definition without changing the name of the security scheme.

### CODE REVIEW

**Comment by Mariangela** [comment]
@Chamindu36 , pls let me know when the WUM fix is released and also details of the WUM fix such as date it was released as that information needs to be added to the document.

**Comment by Chamindu36** [comment]
Sure. I’ll do that

File: en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md [hunk 1]
```diff
@@ -129,14 +129,14 @@ WSO2 API Controller, **apictl** allows to create and deploy APIs without using W
...
                             scopes: 
-                                products:read: ""
+                                read: ""
                             x-scopes-bindings: 
-                                products:read: "admin"
+                                read: "admin"
                 ```
 
-            First you need to define the scope name (products:read) under `security > default` section inside the required resource and then define the role binding under the `securitySchemes` section.
+            First you need to define the scope name ("read") under `security > default` section inside the required resource and then define the role binding under the `securitySchemes` section. You can use your choice of name as the security scheme.
```
Comment (npamudika) [inline en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md:138]: Let's edit as below.
```suggestion
You can use any preferred name as the security scheme.
```

File: en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md [hunk 2]
```diff
@@ -129,14 +129,14 @@ WSO2 API Controller, **apictl** allows to create and deploy APIs without using W
...
 
-            Make sure to set the security name as **`default`** when defining the scopes.        
+            Make sure to set the security type as **`oauth2`** when defining the scopes and when defining roles under a particular scope make sure that roles are include in **x-scopes-bindings:**  then scope name and the role permitted by the scope.
```
Comment (npamudika) [inline en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md:140]: Shall we edit like this?
```suggestion
Make sure to set the security type as **`oauth2`** when defining the scopes. Also when defining the roles under a particular scope, put them under **x-scopes-bindings:**  as a scope name and roles mapping.
```

**Pull request merged**